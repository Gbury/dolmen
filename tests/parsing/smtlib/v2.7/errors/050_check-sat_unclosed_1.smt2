(set-logic ALL)
(check-sat _)
