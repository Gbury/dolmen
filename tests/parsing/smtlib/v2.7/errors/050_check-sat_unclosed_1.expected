File "tests/parsing/smtlib/v2.7/errors/050_check-sat_unclosed_1.smt2", line 2, character 11-12:
2 | (check-sat _)
               ^
Error while parsing a check-sat command, read the reserved word '_',
      but expected a closing parenthesis.
